src/buttons/Button.js

```diff
@@ -12,6 +12,8 @@ import {
 } from 'react-native';
 import colors from '../config/colors';
 
+import colors from '../config/colors';
```

Comment:
This is a duplicate line, colors is already imported on line 13

Reply:
My Bad. Didnt realize that. ;)